source/support.tex

```diff
@@ -1997,7 +1993,11 @@
 respectively.
 \end{note}
 
-\rSec1[stdfloat.syn]{Header \tcode{<stdfloat>} synopsis}
+\rSec2[stdfloat.syn]{Header \tcode{<stdfloat>} synopsis}
+
+\pnum
+The header \libheader{stdfloat} defines type aliases for
+the optional extended floating-point types\iref{basic.extended.fp}.
```

Comment:
@jwakely, @dkolsen-pgi: I made up this new introductory wordingl could you please have a look if it's OK?

Reply:
May I suggest a slight change:

> The header <stdfloat> defines type aliases for the optional extended floating-point types with specified formats ([basic.extended.fp]).

or

> The header <stdfloat> defines type aliases for the optional extended floating-point types that are specified in [basic.extended.fp].

There can be other extended floating-point types beyond those specified in [basic.extended.fp] and [stdfloat.syn].  I want to avoid wording that implies that they are the only extended floating-point types.

Reply:
Good idea, thanks, done!